larq/callbacks.py

```diff
@@ -31,14 +33,25 @@ def __init__(
         schedule: Callable,
         hyperparameter: str,
         optimizer: Optional[keras.optimizers.Optimizer] = None,
+        update_freq: str = "epoch",
         verbose: int = 0,
+        log_name: Optional[str] = None,
```

Comment:
Can you add a docstring for this (also explaining when you'd pass it)?

Reply:
I thought I did

Reply:
I did, but on the wrong repo. Amazing :sweat_smile: